.github/workflows/main.yml

```diff
@@ -88,9 +56,19 @@ jobs:
         uses: docker/build-push-action@v5
         with:
           push: true
-          tags: ${{ steps.meta.outputs.tags }}
+          tags: "${{ env.IMAGE_NAME }}:${{ github.sha }}"
```

Comment:
important: I strongly recommend you to revert this change.

`github.sha` is a commit id, which mean you are going to push to the repository always new tags, but `steps.meta.outputs.tags` is a variable filled by `docker/build-push-action` that conveys to tags according to the branch/release strategy we follow, please check their docs.

Besides that, a tag should follow https://semver.org/, but `SHA` is not readable and gives us no context at all.

Reply:
I remember having this issue with Amazon and Kubernetes, but we solved it with labels. The tag was still the same, but every new push, we set a label with the commit it. Perhaps Azure also supports it.

In the worse case we can push two tags, the correct one and the commit id tag, but I would recommend you to try first other alternatives. Would you mind checking it?